### PR TITLE
add PERSISTENCE_SINGLE_FILE environment variable

### PR DESCRIPTION
**Pull request by collinsauve**
`utils/persistence.py` already has most of the code available to switch on this, it just is not exposed to be configured by an environment variable.

I used the name as it stood. Since this name will be difficult to change once it's in place you may want to consider inverting this to `USE_MULTIPLE_DUMP_FILES` with the default of `false`.   If you'd like to change the name, happy to resubmit this PR :)

### CODE REVIEW

File: localstack/utils/persistence.py [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
 from abc import ABCMeta, abstractmethod
-from localstack.config import DATA_DIR
+from localstack.config import DATA_DIR, is_env_true
 from localstack.utils.aws import aws_stack
 from localstack.utils.common import to_bytes, to_str
 from localstack.utils.bootstrap import is_api_enabled
 from localstack.services.generic_proxy import ProxyListener
 
-USE_SINGLE_DUMP_FILE = True
+USE_SINGLE_DUMP_FILE = is_env_true('USE_SINGLE_DUMP_FILE')
```
Comment (whummer) [inline localstack/utils/persistence.py:16]: Thanks for this PR @collinsauve 👍 . If we want to make this `True` by default, then I think this should be changed to:
```
USE_SINGLE_DUMP_FILE = is_env_not_false('USE_SINGLE_DUMP_FILE')
```

Comment (collinsauve) [reply localstack/utils/persistence.py:16]: oof.  now I have to rethink how I tested this :suspect: 

Definitely will fix that.  Do you want to keep the env variable name?  If so I will rewrite the commit.

Comment (whummer) [reply localstack/utils/persistence.py:16]: Thanks @collinsauve I'd suggest renaming this to `PERSISTENCE_SINGLE_FILE`, to be a bit more in line with the naming scheme of the other config variables.. Thanks

Comment (whummer) [reply localstack/utils/persistence.py:16]: Btw - let's also add this to `CONFIG_ENV_VARS` in config.py: https://github.com/localstack/localstack/blob/master/localstack/config.py#L227

Comment (collinsauve) [reply localstack/utils/persistence.py]: Sry for the delay @whummer.   Rewrote it with that variable, should be good for another review.

**Pull request merged**